src/pageServices/contentScripts/utils.ts

```diff
@@ -0,0 +1,26 @@
+export const utilityScript = () => {
+  const evaluate = ({ xPath, originJdnHash }: Record<string, string>) => {
```

Comment:
why do we need `originJdnHash` here?

Reply:
to be able to compare in result on the plugin side. when we have several script calls we can't match every call with it's result